src/components/ReadMore/index.jsx

```diff
@@ -18,7 +20,7 @@ const ReadMore = ({ children, postId, picCap = false, readMore = true }) => {
     : text;
 
   return (
-    <>
+    <div className="caption_container">
```

Comment:
please check the console, there's an error `<div> cannot appear as a descendant of <p>` perhaps due to this component being rendered inside a paragraph element.